### ISSUE TITLE
Documentation translations

### ISSUE THREAD

**Issue by timgrossmann** [InstaPy/InstaPy#3601]
#### Once our documentation has been simplified, it would be nice to have the curated guides about setting up InstaPy available in different languages.

Advantages of this feature:
- Users will find it easier to understand what to do
- Bigger audience

Possible ways of achieving this while keeping control:
- Finding people that are interested in contributing on a non-code way. Translation of single documents can be done fairly quickly and independently.

> [Bountysource this Issue](https://www.bountysource.com/issues/67576196-documentation-translations)

--- 

###### Feel free to comment with adjustments and improvements

**Comment by gurenovich** [comment]
yes sounds good! I would like read in Russian language, but I can't help you

**Comment by octavius-1** [comment]
I'm in debt with this repo, so I'm going to offer myself as tribute for spanish translation.

**Comment by timgrossmann** [comment]
@octavius-1 Haha, thank you very much! So awesome!

Since we are currently starting to restructure the docs to be more user friendly, so in the beginning it might only be the documents like https://github.com/timgrossmann/InstaPy/tree/master/docs.
Some of them have nearly no text so that's not too big of a deal, but some of them do. As soon as we are done with the new documentation, having it in multiple languages would be great.

Other than that, having someone translate the article about InstaPy (and later the introduction book) would be probably also help many people!
https://medium.freecodecamp.org/my-open-source-instagram-bot-got-me-2-500-real-followers-for-5-in-server-costs-e40491358340?source=activity---post_recommended

It's your call, though 😉 
Thank you for the offer

**Comment by octavius-1** [comment]
@timgrossmann I can start with those 4 translatable documents in master/docs (BlueOcean, Windows, Docker & Raspberry).
What about the main method usage instructions inside the code section? Or should we keep that in english since all the method parameters are coded in english?
